relentless/potential/potential.py

```diff
@@ -569,36 +639,75 @@ def force(self, pair, potentials):
+
+        Based on the adjustments made to the force, the energy values are also adjusted.
+
+        Parameters
+        ----------
+        u : array_like
+            The binned energy values.
+        f : array_like
+            The binned force values.
+        trim : bool
+            Whether to trim off trailing zeros from the regularized force/energy values,
+            defaults to `True`.
+
+        Returns
+        -------
+        float or `None`
+            The value of r at which trimming trailing zeros is implemented or could be
+            implemented (if trim is False). Returns `None` if trimming cannot be performed.
+        array_like
+            Stacks the r, adjusted u, and adjusted f as columns into a `nx3` array for n bins.
+
+        Raises
+        ------
+        IndexError
+            If the energy array is not the same length as the array of r values.
+        IndexError
+            If the force array is not the same length as the array of r values.
+        UserWarning
+            If rmax is too small to cutoff the potential, i.e. if the force at the end
+            of the potential is larger than fcut.
+
+        """
         if len(u) != len(self.r):
             raise IndexError('Potential must have the same length as r.')
         if len(f) != len(self.r):
             raise IndexError('Force must have the same length as r.')
 
         # find first point from beginning that is within energy tolerance
         if self.fmax is not None:
-            cut = np.argmax(np.abs(f) <= self.fmax)
+            cut = np.argmax(f <= self.fmax)
             if cut > 0:
                 u[:cut] = u[cut] - f[cut]*(self.r[:cut] - self.r[cut])
                 f[:cut] = f[cut]
 
         # find first point from end with sufficient force and cutoff the potential after it
         if self.fcut is not None:
-            flags = np.abs(np.flip(f)) >= self.fcut
+            flags = np.flip(f) >= self.fcut
             cut = len(f)-1 - np.argmax(flags)
-            u -= u[cut]
+            u -= u[cut]                         #would this be considered 'regularizing' the energy?
+            if cut == len(f)-1:
+                if f[-1] > self.fcut:
+                    warnings.warn('rmax is too small to cutoff the potential.', UserWarning)
             if cut < len(f)-1:
                 u[(cut+1):] = 0.
                 f[(cut+1):] = 0.
 
         # trim off trailing zeros
         r = self.r.copy()
-        if trim:
-            flags = np.abs(np.flip(f)) > 0
-            cut = len(f) - np.argmax(flags)
-            if cut < len(f)-1:
+        flags = np.abs(np.flip(f)) > 0
+        cut = len(f) - np.argmax(flags)
+        if cut < len(f)-1:
+            rcut = r[cut]
+            if trim:
                 r = r[:(cut+1)]
                 u = u[:(cut+1)]
                 f = f[:(cut+1)]
+        else:
+            rcut = None
 
-        return np.column_stack((r,u,f))
+        return rcut, np.column_stack((r,u,f))
```

Comment:
Let's flip the return order
```suggestion
        return np.column_stack((r,u,f)), rcut
```
It's a little easier to read if the user wants to ignore the second value like,
```py
pot,_ = tab.regularize_force(...)
```